scss/styles.scss

```diff
@@ -186,3 +186,24 @@
 .django  {
   grid-area: django;
 }
+
+@media (max-width: 1024px) {
+  .tech-container {
```

Comment:
This is not wrong but is preferred to avoid an `inline` display. Either modify things to keep grid and make it use flexbox with `flex-direction: column`